test/MoveAdd.lean

```diff
@@ -0,0 +1,98 @@
+import Mathlib.Tactic.MoveAdd
+import Mathlib.Data.Nat.Basic
+
+universe u
+
+section tactic
+open Mathlib.MoveAdd
+
+run_cmd if uniquify [ 0,      1,      0,      1,      0,      3] ≠
```

Comment:
Perhaps move these "internal" tests to the bottom of the file? Test files are effectively part of the documentation, and its helpful if users can learn how to use the tactic by reading the top of the test file!

Reply:
I moved these to the other file.